scripts/release/strip-appimage-graphics-libs.sh

```
@@ -363,6 +363,133 @@ validate_sharun_lib_path() {
+        esac
+        case "+${seen}+" in
+          *"+${entry}+"*) IFS=':'; continue ;;
+        esac
+        seen="${seen}+${entry}"
+        kept+=("$entry")
+        IFS=':'
+      done
+      IFS="$old_ifs"
+
+      if [ "${#kept[@]}" -eq 0 ]; then
+        cleaned='$ORIGIN:$ORIGIN/../shared/lib'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)